frontend/package.json

```diff
@@ -1,36 +1,38 @@
 {
-  "name": "~TODO~",
-  "version": "0.0.1",
-  "scripts": {
-    "dev": "svelte-kit dev",
-    "build": "svelte-kit build",
-    "preview": "svelte-kit preview",
-    "lint": "prettier --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
-    "format": "prettier --write --plugin-search-dir=. ."
-  },
-  "devDependencies": {
-    "@sveltejs/kit": "next",
-    "@typescript-eslint/eslint-plugin": "^4.19.0",
-    "@typescript-eslint/parser": "^4.19.0",
-    "autoprefixer": "^10.2.5",
-    "eslint": "^7.22.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-svelte3": "^3.2.0",
-    "postcss": "^8.2.15",
-    "postcss-font-magician": "^3.0.0",
-    "prettier": "~2.2.1",
-    "prettier-plugin-svelte": "^2.2.0",
-    "svelte": "^3.34.0",
-    "svelte-preprocess": "^4.0.0",
-    "tailwindcss": "^2.1.2",
-    "tslib": "^2.0.0",
-    "typescript": "^4.0.0",
-    "vite-plugin-svelte-svg": "^1.0.0"
-  },
-  "type": "module",
-  "dependencies": {
-    "@tailwindcss/typography": "^0.4.0",
-    "svelte-feather-icons": "^3.3.0",
-    "timeago-simple": "^1.2.4"
-  }
+	"name": "feather-website",
+	"version": "0.0.1",
+	"scripts": {
+		"dev": "svelte-kit dev",
+		"build": "svelte-kit build",
+		"preview": "svelte-kit preview",
+		"lint": "prettier --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
+		"format": "prettier --write --plugin-search-dir=. ."
+	},
+	"devDependencies": {
+		"@sveltejs/kit": "next",
+		"@typescript-eslint/eslint-plugin": "^4.19.0",
+		"@typescript-eslint/parser": "^4.19.0",
+		"autoprefixer": "^10.2.5",
+		"eslint": "^7.22.0",
+		"eslint-config-prettier": "^8.1.0",
+		"eslint-plugin-svelte3": "^3.2.0",
+		"postcss": "^8.2.15",
+		"postcss-font-magician": "^3.0.0",
+		"postcss-load-config": "^3.0.1",
+		"prettier": "~2.2.1",
+		"prettier-plugin-svelte": "^2.2.0",
+		"svelte": "^3.34.0",
+		"svelte-preprocess": "^4.0.0",
+		"tailwindcss": "^2.1.2",
+		"tslib": "^2.0.0",
+		"typescript": "^4.0.0",
+		"vite-plugin-svelte-svg": "^1.0.0"
+	},
+	"type": "module",
+	"dependencies": {
```

Comment:
I think they should all be dev, also we might want to use tailwindcss-forms? Also maybe upgrade our tailwindcss to jit and use https://github.com/svelte-add/tailwindcss